vulnerabilities/models.py

```diff
@@ -1315,7 +1316,7 @@ class Advisory(models.Model):
     """
 
     unique_content_id = models.CharField(
-        max_length=32,
+        max_length=64,
```

Comment:
We need a help text and including an explanation of why we use 64 chars, e.h. sha256 as hex.